change/@fluentui-contrib-azure-theme-9b208281-4f86-407c-994f-9aaf773172c8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
@adr1-jpg I do not think it reflects the changes you made, let's have it as just 'dependentChangeType: none' and 'type: none'